### PR TITLE
Exit code fix

### PR DESCRIPTION
**Pull request by ejthurgo**
Fixes exit codes when operations are successful.

### CODE REVIEW

File: waflyctl.go [hunk 1]
```diff
@@ -2434,10 +2434,10 @@ func main() {
 		//validate the config
 		validateVersion(*client, *serviceID, latest.Number)
 		Info.Println("Completed")
-		os.Exit(1)
+		os.Exit(0)
 	} else {
 		Info.Println("Nothing to do. Exiting")
-		os.Exit(1)
+		os.Exit(0)
```
Comment (fgsch) [inline waflyctl.go]: This is actually an error and should exit with 1.

Comment (ejthurgo) [reply waflyctl.go]: Ah, considering the above msg outputs to the Info log, rather than error. I didn't read this as an error case.

Comment (ejthurgo) [reply waflyctl.go]: But sure, I can change this. I do agree that it should probably be an error.
Should I tweak the log line?

Comment (fgsch) [reply waflyctl.go]: Good idea. 👍

**Pull request merged**